packages/cqrs/lib/src/cqrs.dart

```diff
@@ -166,4 +311,47 @@ class Cqrs {
       },
     ).timeout(_timeout);
   }
+
+  void _log(
+    CqrsMethod method,
+    _ResultType result, [
+    Object? error,
+    StackTrace? stackTrace,
+    List<ValidationError> validationErrors = const [],
+  ]) {
+    if (_logger == null) {
+      return;
+    }
+
+    final log = switch (result) {
+      _ResultType.success => _logger?.info,
+      _ResultType.validationError => _logger?.warning,
+      _ResultType.jsonError ||
+      _ResultType.networkError ||
+      _ResultType.authenticationError ||
+      _ResultType.forbiddenAccessError ||
+      _ResultType.unknownError =>
+        _logger?.severe,
+    };
+
+    final methodTypePrefix = switch (method) {
+      Query() => 'Query',
+      Command() => 'Command',
+      Operation() || _ => 'Operation',
```

Comment:
Perhaps we should mark `CqrsMethod` as sealed to avoid this catchall?

Reply:
Yes, I reverted this by mistake